### ISSUE TITLE
Thunderbird 115: NVDA table navigation commands fail in the list of messages

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#15162]
### Steps to reproduce:
1. Launch NVDA.
2. Install and launch Thunderbird 115.
3. Add at least one e-mail account (with messages present in it) in Thunderbird, so you can browse through folders and messages.
4. Press F6 a few times, to get to the list/table view with the list of messages.
5. Use Up/Down arrows to select a message.
6. Use NVDA's table navigation commands to navigate between the columns for the selected message.

### Actual behavior:
After step 6, NVDA stays silent.

### Expected behavior:
After step 6, NVDA reads the values of each column in the table view for the selected message, when the focus is moved to the respective column.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
NVDA alpha-28694,c4ca763b, in bulgarian.

#### Windows version:
Windows 11 Pro version 22H2 (build 22621.1992), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".

#### Name and version of other software in use when reproducing the issue:
Thunderbird 115.0.0, 64-bit, in bulgarian.

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2023.1 - the result is the sane.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, it is.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes, it does.

**Comment by amirsol81** [comment]
Also duplicable with Thunderbird V116 Beta 3. Interestingly, NVDA does indicate that we're in a table but those commands don't work.

**Comment by amirsol81** [comment]
Also duplicable with the new TB 116 Beta 4.

**Comment by cary-rowen** [comment]
Hi,
This is not an NVDA issue, I have reported it to mozilla.
https://bugzilla.mozilla.org/show_bug.cgi?id=1843486

**Comment by emitche** [comment]
Thank you so much for this report.

At Thunderbird, we thought this might be on NVDA's side, not on Thunderbird's end. We made some significant changes to the code for messages between our 102 and our recent ESR release 115. I am looking into this on Thunderbird's end and will send an update ASAP. This may require coordination with Thunderbird and NV Access.

Thank you!

**Comment by emitche** [comment]
This looks like something we need to change on Thunderbird's end. Thank you!

**Comment by emitche** [comment]
This is fixed with [Thunderbird Bug 1843486](https://bugzilla.mozilla.org/show_bug.cgi?id=1843486).

You can now use the NVDA key and up, down, left, and right to navigate the rows and cells in the Table View of the message list. This is currently available in Thunderbird Daily and is scheduled to be released into Thunderbird Beta this week.

There is a larger effort as part of [Thunderbird Bug 1854787](https://bugzilla.mozilla.org/show_bug.cgi?id=1854787) to make the content within the cells have readable information.

**Comment by Adriani90** [comment]
@emitche thanky very much for the solution on this one, this is very appreciated.
